iteration/iteration.go

```diff
@@ -181,6 +182,35 @@ func (m *GormIterationRepository) CanStartIteration(ctx context.Context, i *Iter
 	return true, nil
 }
 
+// List returns backlog items where iteration is in root iteration and status != closed
+func (m *GormIterationRepository) ListBacklogIterations(ctx context.Context, spaceID uuid.UUID, start *int, limit *int) ([]*Iteration, error) {
```

Comment:
? I think there is a missunderstanding here somewhere..

the 'backlog' link should point to a []WorkItem
where iteration == defaultIteration
and state != closed